src/domain/kaivuilmoitusMuutosilmoitus/KaivuilmoitusMuutosilmoitus.test.tsx

```diff
@@ -144,6 +215,129 @@ describe('Canceling muutosilmoitus', () => {
   });
 });
 
+describe('Muutosilmoitus attachments', () => {
+  async function uploadAttachmentMock({
+    muutosilmoitusId,
+    attachmentType,
+    file,
+    abortSignal,
+  }: {
+    muutosilmoitusId: string;
+    attachmentType: AttachmentType;
+    file: File;
+    abortSignal?: AbortSignal;
+  }) {
+    const { data } = await api.post<MuutosilmoitusAttachmentMetadata>(
+      `/muutosilmoitukset/${muutosilmoitusId}/liitteet?tyyppi=${attachmentType}`,
+      { liite: file },
+      {
+        signal: abortSignal,
+      },
+    );
+    return data;
+  }
+
+  function initFileGetResponse(response: ApplicationAttachmentMetadata[]) {
+    server.use(
+      http.get('/api/hakemukset/:id/liitteet', async () => {
+        return HttpResponse.json(response);
+      }),
+    );
+  }
+
+  test('Should be able to upload attachments', async () => {
```

Comment:
@markohaarni I cannot find the reason why this and the next text fail (`[MSW] Cannot bypass a request when using the "error" strategy for the "onUnhandledRequest" option`). I have used here the exact same testing that is done with täydennys, but for some reason these tests fail.

Reply:
Found the error. I forgot to add the api calls to handlers.ts as well.